src/BotwinExtensions.cs

```diff
@@ -44,19 +40,24 @@ public static IApplicationBuilder UseBotwin(this IApplicationBuilder builder, Bo
                 foreach (var module in scope.ServiceProvider.GetServices<BotwinModule>())
                 {
                     var moduleType = module.GetType();
-                    var distinctPaths = module.Routes.Keys.Select(route => route.path).Distinct();
+
+                    var distinctPaths = module.Routes.Keys.Select(route =>
+                    {
+                        logger?.LogTrace($"Found and adding {moduleType.Name} with {route.verb} /{route.path} to routing");
```

Comment:
This log should be done with the `MapRoute` call below. Collecting distinct paths does actually nothing about the route. It’s the `MapRoute` that does what this log says what happens.

Also, use structured logging instead of string interpolation.